MapboxDirections/MBDirections.swift

```diff
@@ -183,6 +183,26 @@ open class Directions: NSObject {
         return task
     }
     
+    @objc(calculateMatchingWithOptions:completionHandler:)
+    @discardableResult open func calculate(_ options: MatchOptions, completionHandler: @escaping CompletionHandler) -> URLSessionDataTask {
```

Comment:
“Calculate matching” sounds a bit contrived. The existing `calculate(_:completionHandler:)` is so named because “route” can easily be confused with a noun. I think we can call this method `match(_:completionHandler:)` in Swift and `-matchWithOptions:completionHandler:` in Objective-C, but I don’t have a strong opinion on this.